README.md

```diff
@@ -1,34 +1,275 @@
 # markbind-action
-
 A GitHub Action that builds and deploys your MarkBind site.
 
-## How to use this action?
+## Option Summary
+
+Option        | Required |                      Default | Remarks
+:-------------|:--------:|-----------------------------:|-----------------------------------------------------------------------------------------------------------
+token         |   yes    |                              | The token to be used for the service
```

Comment:
Really minor suggestion: Would it be better to make the `Option`s link to their respective `Option Details` section? I think this can help the users to quickly look at the detailed description to get the possible values etc.